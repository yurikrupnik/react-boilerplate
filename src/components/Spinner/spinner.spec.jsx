import React from 'react';
import Component from './index';

const {
    describe,
    it,
    expect,
    shallow,
} = global;

describe(`${Component.name}`, () => {
    it('render', () => {
        const wrapper = shallow(
            <Component />
        );
        expect(wrapper).toMatchSnapshot();
    });

    // it('render with data', () => {
    //     const wrapper = shallow(
    //         <Component
    //             data={[
    //                 {
    //                     _id: 'asd',
    //                     name: 'asd'
    //                 },
    //                 {
    //                     _id: 'asd',
    //                     name: 'asd'
    //                 }
    //             ]}
    //             loading={false}
    //         />
    //     );
    //     // expect(wrapper.length).toBe(1);
    //     expect(wrapper).toMatchSnapshot();
    // });
});
