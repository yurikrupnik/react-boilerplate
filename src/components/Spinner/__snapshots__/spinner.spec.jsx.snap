// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_default render 1`] = `
<div
  style={
    Object {
      "background": "red",
    }
  }
>
  loading...
</div>
`;
